plutus-tx-plugin/src/PlutusTx/Options.hs

```diff
@@ -230,6 +232,11 @@ pluginOptions =
     , let k = "simplifier-beta"
           desc = "Run a simplification pass that performs beta transformations"
        in (k, PluginOption typeRep (setTrue k) posDoSimplifierBeta desc [])
+    , let k = "simplifier-evaluate-builtins"
+          desc =
+            "Run a simplification pass that evaluates fully saturated builtin applications. \
```

Comment:
Please add some comments! To the pass itself and somewhere here, so that the user knows they are enabling a semantics-changing pass. At least the link to the issue.